### PR TITLE
Gonzus/c library benchmark

### PR DESCRIPTION
**Pull request by gonzus**
Add a benchmark to compare C and C++ versions
Make benchmark output easily parseable
Make benchmark run on linux
Keep benchmark cases sorted alphabetically

### CODE REVIEW

File: c/Makefile [hunk 1]
```diff
@@ -30,6 +31,9 @@ $(LIB_A_FILE): $(LIB_OBJ_FILES)
 example: example.o $(LIB_A_FILE)
 	$(CC) $(ALL_FLAGS) -o $@ example.o $(LD_FLAGS) -L. -l$(LIB_NAME)
 
+gonzo: gonzo.o $(LIB_A_FILE)
```
Comment (zongweil) [inline c/Makefile]: I don't think you want to include this here :).

File: c/gonzo.c [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
```
Comment (zongweil) [inline c/gonzo.c]: Please leave this file off.

File: c/Makefile (hunk 1 above)
Comment (gonzus) [reply c/Makefile]: Good catch, removed with b430c289e54354d3116a60dfdfac8d487d2b6cc7.

File: c/gonzo.c (hunk 2 above)
Comment (gonzus) [reply c/gonzo.c]: Good catch, removed with b430c289e54354d3116a60dfdfac8d487d2b6cc7.

**Comment by drinckes** [comment]
No other implementation has a benchmark, so what is the justification for having this specific one checked into the repo? I don't expect that it's something that we're going to want to run regularly, or am I missing something?

**Comment by gonzus** [comment]
You are correct. On the other hand, benchmarks are usually included in a project's repo for transparency, so that anyone can replicate the measurements and come to their own conclusions.

I am OK with not having this here, of course.

**Comment by gonzus** [comment]
I am not sure if other changes are required, or if this is still under consideration (which is fine, of course)... I ask because I keep seeing a "Changes requested" and I don't have any additional way for me to say "your requested changes were applied".

**Comment by zongweil** [comment]
Sorry, I didn't get an email notification with your code changes (and I was out for Thanksgiving). Code looks good and I don't have any concerns with including this here, so merging now.

**Pull request merged**